example/app/controllers/application_controller.rb

```diff
@@ -1,5 +1,5 @@
 class ApplicationController < ActionController::Base
-  include ShopifyApp::Controller
+  include ShopifyApp::LoginProtection
```

Comment:
thanks for updating this too. I have another PR incoming since the example needs an update after the name-spacing.